# DetectedObject message
# This structure contains the information of a detected object.

# Object type
string type

# Whether or not the object is faulty
bool is_faulty

# Target pose
geometry_msgs/Pose pose
